libdnf/module/ModulePackageContainer.cpp

```diff
@@ -742,7 +749,11 @@ void ModulePackageContainer::enableDependencyTree(std::vector<ModulePackage *> &
 ModulePackageContainer::ModuleState
 ModulePackageContainer::getModuleState(const std::string& name)
 {
-    return pImpl->persistor->getState(name);
+    try {
+        return pImpl->persistor->getState(name);
+    } catch (NoModuleException &) {
+        return ModuleState::UNKNOWN;
```

Comment:
Same thing here, although returning `UNKNOWN` might seem better, it really is not, because we still haven't found the module, which is different, and we shouldn't bundle multiple distinct states into a single "we don't know anything" state.